packages/bpk-component-drawer/src/BpkDrawer.js

```diff
@@ -118,6 +119,7 @@ BpkDrawer.defaultProps = {
   closeLabel: null,
   closeText: null,
   hideTitle: false,
+  padded: true,
```

Comment:
New option - default to `true` for backwards compatibility. Consumers can opt-out for new non-padded behaviour.